ui/pages/send/send.test.js

```diff
@@ -146,7 +146,7 @@ describe('Send Page', () => {
     });
   });
 
-  describe('Send and Edit Flow', () => {
+  describe('Send and Edit Flow (draft)', () => {
```

Comment:
I am not sure what the "draft" stage is or how to reproduce it by hand. I tacked "(draft)" onto the end of this `describe` to differentiate it from the new wording on line 127 — any thoughts on a better way to word this or is this clear as-is?

Reply:
this works!